pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/basicalloydbsecondarycluster/dependencies.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeNetwork
+metadata:
+  name: computenetwork-${uniqueId}
+spec:
+  autoCreateSubnetworks: false
+---
+apiVersion: alloydb.cnrm.cloud.google.com/v1beta1
+kind: AlloyDBCluster
+metadata:
+  name: alloydbcluster-1-${uniqueId}
+spec:
+  initialUser:
+    password:
+      value: alloydb-pg
+  location: us-west1
+  networkConfig:
+    networkRef:
+      name: computenetwork-${uniqueId}
+  projectRef:
+    external: ${projectId}
+---
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeAddress
+metadata:
+  name: computeaddress-${uniqueId}
+spec:
+  location: global
+  addressType: INTERNAL
+  networkRef:
+    name: computenetwork-${uniqueId}
+  prefixLength: 16
+  purpose: VPC_PEERING
+---
+apiVersion: servicenetworking.cnrm.cloud.google.com/v1beta1
+kind: ServiceNetworkingConnection
+metadata:
+  name: servicenetworkingconnection-${uniqueId}
+spec:
+  networkRef:
+    name: computenetwork-${uniqueId}
+  reservedPeeringRanges:
+  - name: computeaddress-${uniqueId}
+  service: servicenetworking.googleapis.com
+---
+apiVersion: alloydb.cnrm.cloud.google.com/v1beta1
```

Comment:
Just curious: Is this resource necessary?

Reply:
Yes.
As per user journey, the secondary cluster is created after the primary instance gets created in the primary cluster
And we need the servicenetworking and computeaddress to create the primary instance in the dependencies.